builders/scripts/install-libwasmvm.sh

```diff
@@ -5,7 +5,7 @@
 
 set -e
 
-WASMVM_VERSION=`go list -m github.com/Finschia/wasmvm | awk '{print $2}' | grep -o 'v\d\+\.\d\+\.\d\+-\d\+\.\d\+\.\d\+'`
+WASMVM_VERSION=`go list -m github.com/Finschia/wasmvm | awk '{print $2}' | grep -o 'v\d\+\.\d\+\.\d\+-\d\+\.\d\+\.\d\+\(-[a-zA-Z0-9]\+\)\?'`
```

Comment:
Is there a reason you didn't specify "-dynamiclink" in the regular expression here?

Reply:
for other tags, for example "rc1"

Reply:
Understood, thank you!